pokemongo_bot/websocket_remote_control.py

```diff
@@ -0,0 +1,49 @@
+import threading
+from socketIO_client import SocketIO, BaseNamespace
+
+
+class WebsocketRemoteControl(object):
+
+
+    def __init__(self, bot):
+        self.bot = bot
+        self.host, port_str = self.bot.config.websocket_server_url.split(':')
+        self.port = int(port_str)
+        self.sio = SocketIO(self.host, self.port)
+        self.sio.on('bot:process_request', self.on_remote_command)
+        self.thread = threading.Thread(target=self.process_messages)
+
+    def start(self):
+        self.thread.start()
+        return self
+
+    def process_messages(self):
+        while True:
+            self.sio.wait(5)
```

Comment:
are you missing a line in here? something to call on_remote_command or something?